datasets/src/generate.rs

```diff
@@ -66,21 +67,23 @@ fn make_blob(
 /// use linfa_datasets::generate::make_dataset;
 /// let feat_distr = Laplace::new(0.5, 5. ).unwrap();
 /// let target_distr = DiscreteUniform::new(0, 5).unwrap();
-/// make_dataset(5, 5, 2, feat_distr, target_distr);
+/// make_dataset(5, 5, feat_distr, target_distr);
 /// ```
 pub fn make_dataset<X, Y>(
     num_rows: usize,
     num_feats: usize,
-    num_targets: usize,
     feat_distr: X,
     target_distr: Y,
-) -> Dataset<f64, f64>
+) -> DatasetBase<
+    ArrayBase<OwnedRepr<f64>, Dim<[usize; 2]>>,
+    ArrayBase<OwnedRepr<f64>, Dim<[usize; 1]>>,
+>
```

Comment:
You can put `Dataset<f64, f64, Ix1>` instead.

Reply:
will do.